__tests__/index.test.ts

```diff
@@ -347,6 +347,15 @@ describe('Auth0', () => {
         `https://test.auth0.com/authorize?query=params${TEST_TELEMETRY_QUERY_STRING}`
       );
     });
+    it('can be called with no arguments', async () => {
```

Comment:
I'd repeat this test case for the `loginWithPopup` method, and rollback the other changes so you can show me how the old syntax still works somewhere.

Reply:
@lbalmaceda done!